decidim-dev/lib/tasks/locale_checker.rake

```diff
@@ -18,4 +18,14 @@ namespace :decidim do
       end
     end
   end
+
+  desc "Allows a decidim installation to sanitize its available locales"
+  task sanitize_locales: :environment do
+    Decidim::Organization.find_each do |organization|
+      sanitized_locales =
+        organization.available_locales & Decidim.available_locales.map(&:to_s)
```

Comment:
If we configure a reasonable set of locales per organization, I don't think we need this at the moment. I'll remove it, ok?